.github/workflows/pythonbuild.yml

```diff
@@ -259,9 +259,12 @@ jobs:
           uv pip freeze
       - name: Install FlyteCTL
         uses: unionai-oss/flytectl-setup-action@master
+        with:
+          # Remove version and go back to latest when unionai-oss/flytectl-setup-action is fixed
```

Comment:
https://github.com/unionai-oss/flytectl-setup-action/pull/5 is the fix.

Reply:
You can revert this change. The above PR was merged and I tested it in https://github.com/flyteorg/flyte/pull/5377.